src/constants/test/CODATA2014.test.cpp

```diff
@@ -9,106 +9,23 @@
 #include "constants/test/testing.hpp"
 
 using namespace dimwits;
-using namespace njoy;
-
-template< typename T >
-struct echo;
-
-auto mock = addUncertainty(
-  hana::make_map(
-    hana::make_pair( constants::speedOfLight , 2.99792458e8*meter/second )
-  ),
-  hana::make_map(
-    hana::make_pair( constants::speedOfLight , 0.0*meter/second )
-  )
-);
 
 SCENARIO("test all the constants"){
-  // auto data = njoy::constants::CODATA2014;
-  auto data = mock;
+  auto CODATA = CODATA2014;
   auto referenceValues = defineReferenceValues( std::ifstream("CODATA2014.txt") );
 
-  hana::for_each(hana::keys( data ), [&]( auto&& key ){
+  hana::for_each( hana::keys( CODATA ), [&]( auto&& key ){
+    auto refKey = stringFor[ key ];
 
-    auto reference = referenceValues[ stringFor[ key ] ];
+    GIVEN( "constant: " + refKey ){
+      auto reference = referenceValues[ stringFor[ key ] ];
 
-    CHECK( reference.first == data[ key ].value );
-    CHECK( 
-      hana::if_( key ^hana::in^ hana::keys( data.uncertainty ), 
-                reference.second == data.uncertainty[ key ].value,
-                true )
-    );
+      CHECK( reference.first == CODATA[ key ].value );
+      CHECK( 
```

Comment:
```
hana::eval_if(
  hana::not_(key ^hana::in^ hana::keys( CODATA.uncertainty )),
  []{ return true; },
  [](auto identity){ return identity(reference.second == CODATA.uncertainty[ key ].value); });
```

Reply:
This doesn't solve the problem introduced with the `ALIAS` macro.